src/js/controller/viewMixin/week.js

```diff
@@ -219,12 +219,14 @@ var Week = {
      * @param {Date} start start date.
      * @param {Date} end end date.
      * @param {function[]} [andFilters] - optional filters to applying search query
+     * @param {Array.<Objecy[]>} panels - schedule panels like 'milestone', 'task', 'allday', 'time'
```

Comment:
1) `panels` type is `Object[]`.
2) Is there any case that having `panels` but no `andFilter`?

Reply:
1. Thanks a lot.
2. I will change parameter sequence. So optional parameter is last.
```js
/**
 * Populate schedules in date range.
 * @this Base
 * @param {Date} start start date.
 * @param {Date} end end date.
 * @param {Array.<object[]>} panels - schedule panels like 'milestone', 'task', 'allday', 'time'
 * @param {function[]} [andFilters] - optional filters to applying search query
 * @returns {object} schedules grouped by dates.
 */
findByDateRange: function(start, end, panels, andFilters) {
}
```